e2e/tests-dfx/assetscanister.bash

```diff
@@ -459,6 +472,44 @@ check_permission_failure() {
   assert_command_fail dfx canister call e2e_project_frontend commit_proposed_batch "$args" --identity anonymous
   assert_contains "Caller does not have Commit permission"
 
+  # get_configuration()
+  args="()"
+  assert_command      dfx canister call e2e_project_frontend get_configuration "$args" --identity commit
+  assert_command      dfx canister call e2e_project_frontend get_configuration "$args" --identity prepare
+
+  assert_command_fail dfx canister call e2e_project_frontend get_configuration "$args" --identity manage-permissions
+  assert_contains "Caller does not have Prepare permission"
+  assert_command_fail dfx canister call e2e_project_frontend get_configuration "$args" --identity no-permissions
+  assert_contains "Caller does not have Prepare permission"
+  assert_command_fail dfx canister call e2e_project_frontend get_configuration "$args" --identity anonymous
+  assert_contains "Caller does not have Prepare permission"
+
+  # configure()
+  args="(record { })"
+  assert_command      dfx canister call e2e_project_frontend configure "$args" --identity commit
+
+  assert_command_fail dfx canister call e2e_project_frontend configure "$args" --identity prepare
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend configure "$args" --identity manage-permissions
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend configure "$args" --identity no-permissions
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend configure "$args" --identity anonymous
+  assert_contains "Caller does not have Commit permission"
+
+  # validate_configure()
+  args="(record { })"
+  assert_command      dfx canister call e2e_project_frontend validate_configure "$args" --identity commit
+
+  assert_command_fail dfx canister call e2e_project_frontend validate_configure "$args" --identity prepare
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend validate_configure "$args" --identity manage-permissions
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend validate_configure "$args" --identity no-permissions
+  assert_contains "Caller does not have Commit permission"
+  assert_command_fail dfx canister call e2e_project_frontend validate_configure "$args" --identity anonymous
+  assert_contains "Caller does not have Commit permission"
```

Comment:
Should the validate function check if a caller has permissions? I thought it should only validate if the payload is a valid payload to submit to the 'real' function